resources/views/directory/table.blade.php

```diff
@@ -0,0 +1,99 @@
+@extends('committee.members')
+
+@section('panel-heading')
+    <div class="panel-heading">Members Directory</div>
+@endsection
+
+@section('buttons')
+    <div id="columnControls">
+        <label><input type="checkbox" name="name" checked> Name</label>
```

Comment:
I thought that it would make the page a little more interactive. Hmmm do you think there are any disadvantages?